sbi/neural_nets/density_estimators/zuko_flow.py

```diff
@@ -0,0 +1,150 @@
+from typing import Tuple
```

Comment:
suggestion: have one file `density_estimators/flows.py` that contains different flow wrappers. Instead of having one for each backend. Not sure what the better way is here in terms of design...